docs/BR.md

```diff
@@ -1810,28 +1810,28 @@ If the CA asserts compliance with these Baseline Requirements, all certificates
 
 #### 7.1.2.1 Root CA Certificate Profile
```

Comment:
Justification for changes (in this Section):

```
- GitHub-flavored markdown does not support 2^159^ for exponentiation. Consider Unicode SUPERSCRIPT numerals (i.e. 2¹⁵⁹)?

- GitHub-flavored markdown does not support \ \ \ \ inside tables for indentation. Consider Unicode non-breaking figure spaces (i.e.     foo)?```

My opinion: Looks good to me, table appears to render fine.